src/ast/irbuilderbpf.cpp

```diff
@@ -161,12 +161,14 @@ StructType *IRBuilderBPF::GetStackStructType(bool is_ustack)
       getInt32Ty(), // nr_stack_frames
       getInt32Ty(), // pid
       getInt32Ty(), // probe id
+      getInt32Ty(), // __padding
```

Comment:
Just leave an issue # pointing to this PR? Or some other breadcrumb?

@jordalgo I'm pretty sure we just talked about sneaky bits from the "type system mismatch" very recently, do you remember the bug and/or if there's a canonical bug for this generally?

Reply:
I don't think one was filed as this is only happening on earlier kernels but yeah a comment/breadcrumb would be good.

I think this bug was introduced here:
https://github.com/bpftrace/bpftrace/pull/3769